src/WebApiToTypeScript/Endpoints/AngularEndpointsService.cs

```diff
@@ -72,6 +93,13 @@ public void WriteServiceObjectToBlock(TypeScriptBlock serviceBlock, WebApiContro
                     var interfaceWithCallFullName = $"{Config.EndpointsNamespace}.{webApiController.Name}.I{actionName}WithCall";
                     var endpointFullName = $"{Config.EndpointsNamespace}.{webApiController.Name}.{actionName}";
 
+                    var cachedBlock = Config.EndpointsSupportCaching &&
```

Comment:
I'd prefer if we make this a boolean variable first, and then do a `if (boolean) block.AddAndUseBlock` below and get rid of the newly added `AddBlock(block)` methods just for consistency. We could then begin a discussion if we need to support the other format? I dislike null checks. Control flows without null is better I think?